src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java

```diff
@@ -128,6 +128,11 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
             String message = result.getMsg();
             GHCommitState state = result.getState();
 
+            listener.getLogger().printf(
+                    "[GitHub Commit Status Setter] Going to set status %s on repos %s (sha:%7.7s) with context:%s%n",
```

Comment:
Could be renamed to `Set GitHub commit status [universal]` btw and then rename log output without `going blabla` as it already what `Descriptor.getDisplayName()` claim.